<launch>
  <rosparam file="$(find hikrobot_mvs_ros)/config/camera.yaml" command="load" />
  <node pkg="hikrobot_mvs_ros" type="hikrobot_mvs_ros" name="mvs_camera" args="$(find hikrobot_mvs_ros)/config/camera.yaml" respawn="true" output="screen" />
  <node pkg="rviz" type="rviz" name="rviz" args="-d $(find hikrobot_mvs_ros)/rviz_cfg/mvs_camera.rviz" />
</launch>
